metro.config.js

```diff
@@ -1,17 +1,28 @@
+const {getDefaultConfig} = require('metro-config');
+
 /**
  * Metro configuration for React Native
  * https://github.com/facebook/react-native
  *
  * @format
  */
-
-module.exports = {
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: false,
-      },
-    }),
-  },
-};
+module.exports = (async () => {
+  const {
+    resolver: {sourceExts, assetExts}
+  } = await getDefaultConfig();
+  return {
+    transformer: {
+      getTransformOptions: async () => ({
+        transform: {
+          experimentalImportSupport: false,
+          inlineRequires: false
+        }
+      }),
+      babelTransformerPath: require.resolve('react-native-svg-transformer')
+    },
+    resolver: {
+      assetExts: assetExts.filter(ext => ext !== 'svg'),
+      sourceExts: [...sourceExts, 'svg']
+    }
+  };
+})();
```

Comment:
can you explain a little about this change? 
Is it something our users need to do to make this feature work?

Reply:
This is part of the configuration of `react-native-svg-transformer`, this part is needed when loading files.
Yes, they will need to add this if they want to load SVG files.

Reply:
This is part of the SVG library instructions? 
Maybe we can mention it somewhere in the component docs..

Reply:
Yes, added docs.